src/parser.rs

```diff
@@ -365,7 +365,7 @@ impl<'run, 'src> Parser<'run, 'src> {
             let optional = self.accepted(QuestionMark)?;
             let (path, relative) = self.parse_string_literal_token()?;
             items.push(Item::Import {
-              absolute: None,
+              absolute_paths: vec![],
```

Comment:
```suggestion
              absolute_paths: Vec::new(),
```

(No real reason, I just like avoiding macros.)